src/aggregate/aggregate_request.c

```diff
@@ -620,6 +637,14 @@ static int parseGroupby(AREQ *req, ArgsCursor *ac, QueryError *status) {
     return REDISMODULE_ERR;
   }
 
+  for (size_t ii = 0; ii < groupArgs.argc; ++ii) {
+    if (*(char*)groupArgs.objs[ii] != '@') {
+      QERR_MKBADARGS_FMT(status, "Bad arguments for GROUPBY: Unknown property `%s`. Did you mean `@%s`?",
+                         groupArgs.objs[ii], groupArgs.objs[ii]);
+      return REDISMODULE_ERR;
+    }
+  }
```

Comment:
in the `GROUPBY` step we just skipped the first character, while in other places (LOAD, REDUCE) we ignore it only if it's an @. we can do that as well instead, it just feels a bit odd.

Reply:
Agree it's better to be consistent.
Skipping the fist character can be done if it is guaranteed that it is a `@`.
If not, it  might mistakenly refer to another field, e.g., if an `a` field exists, and an `aa` is used by the grouper.
(BTW skipping the first is also not suitable in case we would like to allow both schema fields and also non-schema fields, so the former begin with `@` and the latter do not)